src/test/java/com/aws/greengrass/clientdevices/auth/metrics/MetricsTest.java

```diff
@@ -367,4 +367,11 @@ void GIVEN_serviceErrorEvent_WHEN_eventsEmitted_THEN_serviceErrorMetricCorrectly
         assertEquals(metric.getUnit(), serviceError.getUnit());
         assertEquals(metric.getNamespace(), serviceError.getNamespace());
     }
+
+    @Test
+    void placeholder() {
```

Comment:
placeholder? still needed?

If needed, give it a better name.

Reply:
Ah, nope, not needed - I was just using it to try some things out and forgot to remove it